intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -8,15 +10,22 @@ editor_options:
   chunk_output_type: inline
 ---
 
-**CCDL 2020**
 
-## Objective for this notebook analysis: 
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Use the tidyverse package to read and write data frames
+- Use dplyr pipes to manipulate data frames
+- Implement and use functions in the tidyverse package to wrangle data (i.e. filter, mutate, arrange, join)
+- Use the apply function to apply functions across entire data frames  
```

Comment:
I don't love the redundancy, but I can't figure out a way around it! 
I also changed style here to make the function name stand out, and highlighted the fact that we are using it specifically with matrices here.
```suggestion
- Use the `apply()` function to apply functions across rows or columns of a matrix
```